lib/handler.js

```diff
@@ -1,8 +1,7 @@
-const _ = require('lodash');
-const aeclient = require('aws-elasticsearch-client');
-const AWS = require('aws-sdk');
 const bunyan = require('alpha-lambda-bunyan');
+const DynamoDB = require('aws-sdk/clients/dynamodb');
 const lambdaHandler = require('alpha-lambda');
+const pick = require('lodash/pick');
```

Comment:
Those packages don't appear to be supported anymore. Most of them haven't had a release in 3 years, but lodash has had many security vulnerabilities pop up in since then, so I don't trust the lodash.* packages.

https://www.npmjs.com/search?q=keywords:lodash-modularized

Reply:
They have not had released in 3 years because these specific function had no updates. Per-function publishing [is supported indeed](lodash.com).

Reply:
That link doesn't work?